config/vufind/reserves.ini

```diff
@@ -42,6 +42,30 @@ formatting_rule[*] = "phrase"
 [Autocomplete_Types]
 Reserves = "SolrReserves:AllFields:course,instructor,department"
 
+[Advanced_Settings]
+; Any facet fields named in the translated_facets[] list below will have their values
+; run through the translation code; values from unlisted facet fields will be
+; displayed as-is without translation.
+;
+; For translated facets, be sure that all of the necessary strings are included in the
+; language files found in the languages directory.
+;
+; You may add a colon and the name of a text domain after the field name to specify
+; translations in a specific text domain (subdirectory of the languages folder).
+;
+; If you add a second colon, you may add a  translation string containing the
+; placeholders %%raw%% and %%translated%%. You can enter the translation string
+; directly (if the formatting is uniform for all languages), or you can use a key
+; pointing to values found in the language files (if you need different formatting
+; for different languages). This can be useful if the facet is related to a
+; classification and you would like to show the raw value together with the
+; translation (see the Dewey example below for a possible application).
+;
+; Commenting out all translated_facets[] lines below will disable facet translation.
```

Comment:
You can remove all these comments, since the comment at the top of the file refers to the other configs that contain the documentation (to prevent things from getting out of sync and causing confusion).

```suggestion
```

Reply:
Makes sense -- I should have caught that. Thanks! Just committed that.